src/protocols/bdx/AsyncTransferFacilitator.cpp

```diff
@@ -194,6 +191,8 @@ void AsyncResponder::NotifyEventHandled(const TransferSession::OutputEventType e
         eventType == TransferSession::OutputEventType::kTransferTimeout ||
         eventType == TransferSession::OutputEventType::kStatusReceived)
     {
+        ChipLogDetail(BDX, "NotifyEventHandled : Event %s Error %" CHIP_ERROR_FORMAT,
```

Comment:
If we only do this for transfer-terminating events, it should be ChipLogProgress.

Reply:
changed to ChipLogProgress